content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -135,8 +140,7 @@ spec:
 
 This pod runs in the `Burstable` QoS class because requests are less than limits.
 
-If the selected policy is anything other than `none` , Topology Manager would not consider either of these Pod
-specifications. 
+If the selected policy is anything other than `none`, Topology Manager would consider either of these Pod specifications. The Topology Manager would consult the Hint Provider, which is CPU Manager to get topology hints. In the case of the `static` CPU Manager policy would return default topology hint, because either of these Pods do not have CPU request explicitly.

```

Comment:
Perhaps reword to something like:

If the selected policy is anything other than `none`, Topology Manager would consider these Pod specifications. The Topology Manager would consult the Hint Providers to get topology hints. In the case of the `static` CPU Manager policy, the default topology hint would be selected, because these Pods do not have explicity request CPU resources.

Reply:
updated